### PR TITLE
[TRIVIAL] Add some missing items to help command list:

### PR DESCRIPTION
**Pull request by ximinez**
* validators
* validator_list_sites
* Put "version" in the right place

Since these commands ignore parameters, there's not much to it.

### CODE REVIEW

File: src/ripple/app/main/Main.cpp [hunk 1]
```diff
@@ -161,6 +160,9 @@ void printHelp (const po::options_description& desc)
            "     submit_multisigned <tx_json>\n"
            "     tx <id>\n"
            "     validation_create [<seed>|<pass_phrase>|<key>]\n"
+           "     validators"
+           "     validator_list_sites"
+           "     version\n"
            "     wallet_propose [<passphrase>]\n";
 }
```
Comment (mellery451) [inline src/ripple/app/main/Main.cpp]: this looks good, but seems like it still doesn't completely match the list at https://github.com/ripple/rippled/blob/5214b3c1b09749420ed0682a2e49fbbd759741e5/src/ripple/rpc/impl/Handler.cpp#L58-L116  -- should it or do we deliberately omit some ?

Comment (nbougalis) [reply src/ripple/app/main/Main.cpp]: We deliberately omit some of these.

File: src/ripple/app/main/Main.cpp [hunk 2]
```diff
@@ -161,6 +160,9 @@ void printHelp (const po::options_description& desc)
            "     submit_multisigned <tx_json>\n"
            "     tx <id>\n"
            "     validation_create [<seed>|<pass_phrase>|<key>]\n"
+           "     validators"
```
Comment (MarkusTeufelberger) [inline src/ripple/app/main/Main.cpp]:
```suggestion
           "     validators\n"
```

File: src/ripple/app/main/Main.cpp [hunk 3]
```diff
@@ -161,6 +160,9 @@ void printHelp (const po::options_description& desc)
...
+           "     validator_list_sites"
```
Comment (MarkusTeufelberger) [inline src/ripple/app/main/Main.cpp]:
```suggestion
           "     validator_list_sites\n"
```

File: src/ripple/app/main/Main.cpp (hunk 1 above)
Comment (ximinez) [reply src/ripple/app/main/Main.cpp]: Which ones do we deliberately omit? Should we make a notation in `handlerArray` as such?

**Comment by MarkusTeufelberger** [comment]
Oh damn it, my company mail address seems to be used by default in these "suggested changes"... O_o

GitHub still manages to put in slightly annoying things for people working on open source both privately and professionally... I'm sure my employer is ok with me committing 2 newlines though, so no biggie.

**Comment by ximinez** [comment]
@MarkusTeufelberger I squashed the commits.

File: src/ripple/app/main/Main.cpp (hunk 1 above)
Comment (nbougalis) [reply src/ripple/app/main/Main.cpp]: Generally, the ones that require multiple parameters or involve complicated parsing.

**Comment by nbougalis** [comment]
@MarkusTeufelberger: you can choose which e-mail address to use when making changes via the GitHub web-based UI (I'm assuming you use it for your commit)

**Pull request closed without merging**